### DISCUSSION TITLE
How to configure the combo display

### DISCUSSION THREAD

**Discussion by drgarys** [artisan-roaster-scope/artisan#944] [Q&A] [answered]
I'm running Artisan 2.6.0 on a Mac that runs Mac OS 12.5.1. My display barely shows the stepped events at the bottom and doesn't create much height difference in the steps. Is there a way to adjust the display so it's more readable?
[Colombia Huila graph for Artisan question](https://user-images.githubusercontent.com/48734972/187269291-55c53b90-9365-4975-b42f-cd0f6333f148.png)

**Comment by MAKOMO** [comment, marked as answer]
By default the 100% step is drawn below the lowest phase limit (Drying min under Config >> Phases). You can raise that phases limit to raise the 100% step.

If "Snap" under Config >> Events, first tab is ticked, the 0% step is aligned with the 0° y-axis and 
 the 100% step with the 100° y-axis temperature values.

Finally, the 100% step value can be set independent of the Phases limits and the temperature/y-axis 0/100° values, by putting a temperature degree to be used as the 100% value into the box "100% Event Step" in the axis dialog (Config >> Axis). If a value is set in this box, the 0% step is aligned with the min y-axis temperature value.

While it sounds confusing, there are those 3 ways to define the invisible events y-axis in rather flexible ways.

**Reply by drgarys** [reply]
Thanks, Marko. I tried to follow those instructions as best I could, and you're right, it's confusing. The step markings are still not very visible, so I must not quite be getting it. I'll paste in some screen shots so you can see where I'm off. Please let me know if there are other screen shots that would help.

![Screen Shot 2022-08-30 at 6 39 57 PM](https://user-images.githubusercontent.com/48734972/187574243-64aaa5b0-d900-4f5e-9c97-4ceb058deede.png)
![Screen Shot 2022-08-30 at 6 39 31 PM](https://user-images.githubusercontent.com/48734972/187574264-315ac959-ba12-4e47-8128-a2d4a0505b5c.png)
![Screen Shot 2022-08-30 at 6 38 32 PM](https://user-images.githubusercontent.com/48734972/187574298-d1f764ad-d8d6-472c-859e-d61fdfa446e6.png)
![Screen Shot 2022-08-30 at 6 38 12 PM](https://user-images.githubusercontent.com/48734972/187574317-5f10f52c-a6a8-4406-8873-56282e818e82.png)

**Reply by poundy** [reply]
You must still have this set: 
|  If "Snap" under Config >> Events, first tab is ticked, the 0% step is aligned with the 0° y-axis and
|  the 100% step with the 100° y-axis temperature values.
because the numbers look to be on the 0 to 100 degree scale.  That's the dialog box you didn't print-screen :) 
Plus I also notice you still oave 100 in the "100% event step". Set that to 300 and see if that alone addresses it?

**Reply by MAKOMO** [reply]
Your 100% step is set to 100, just put it to 200 or wherever you like it under menu Config >> Axis.

**Reply by drgarys** [reply]
Thanks, both. I've tried what you've suggested and it doesn't seem to work yet. Here are some more printed screens. I'm sure there's a way to do this and I'm not quite finding the setting.

![Screen Shot 2022-08-31 at 6 22 32 AM](https://user-images.githubusercontent.com/48734972/187689710-02b58cbd-674a-4e88-adc3-d206cf132212.png)
![Screen Shot 2022-08-31 at 6 23 40 AM](https://user-images.githubusercontent.com/48734972/187689748-cfbd7642-1ab3-487d-a1d7-678e7e5bf467.png)
![Screen Shot 2022-08-31 at 6 24 20 AM](https://user-images.githubusercontent.com/48734972/187689770-383e0385-e43b-4ddd-94da-d5f783fb8c34.png)

I wonder if the buttons are getting in the way below the roast chart? Nope. That didn't reveal more of the stepped display.

**Reply by MAKOMO** [reply]
You still have "Snap" ticked in the Events dialog, thus the events are "snapped" to the y-axis with 0% at 0° and 100% at 100° as I described before. Just untick that one.

**Reply by drgarys** [reply]
That worked! It's often just one setting away. 

![Screen Shot 2022-08-31 at 11 33 35 AM](https://user-images.githubusercontent.com/48734972/187753949-188af6a3-8a13-4a0f-aa02-753f97f55479.png)
